fpm/recipes/jenkins-agent/recipe.rb

```diff
@@ -0,0 +1,21 @@
+class Gof3r < FPM::Cookery::Recipe
+  description "The Jenkins swarm plugin enables slaves to auto-discover nearby Jenkins master and join it automatically, thereby forming an ad-hoc cluster."
```

Comment:
This string doesn't have any variables so it should be single quoted to make linting tools happy.